packages/perseus/src/widgets/interactive-graphs/graphs/angle.tsx

```diff
@@ -95,9 +97,33 @@ function AngleGraph(props: AngleGraphProps) {
         showAngles: showAngles || false, // Whether to show the angle or not
     };
 
+    const {strings, locale} = usePerseusI18n();
+
+    const angleMeasure = srFormatNumber(
+        getClockwiseAngle(
+            [endPoints[0], centerPoint, endPoints[1]],
+            allowReflexAngles,
+        ),
+        locale,
+    );
+
+    const wholeAngleAriaLabel = strings.srAngleGraphAriaLabel;
+    const wholeAngleDescription = strings.srAngleGraphAriaDescription({
+        angleMeasure,
+        vertexX: coords[1][X],
+        vertexY: coords[1][Y],
+        isX: coords[2][X],
+        isY: coords[2][Y],
+        tsX: coords[0][X],
+        tsY: coords[0][Y],
```

Comment:
I think you need to wrap all of these in `srFormatNumber()` so they truncate and react correctly.